### ISSUE TITLE
megaboilerplate.com not working

### ISSUE THREAD

**Issue by matheuschimelli** [sahat/megaboilerplate#222]
![image](https://user-images.githubusercontent.com/10470074/39140319-94444f7a-46fa-11e8-82ac-0a0e1547e158.png)

**Comment by KayvanMazaheri** [comment]
**megaboilerplate.com is unavailable.**

<img width="1440" alt="megaboilerplate.com is unavailable." src="https://user-images.githubusercontent.com/5472278/40667726-11a394b4-6378-11e8-9566-f84786d54671.png">

**Comment by matheuschimelli** [comment]
I been thinking, that this project maybe is dead. I got run it on heroku.
If you want to use : https://rocket-generator.herokuapp.com . Only zip
download is working.

**Comment by erickbrodri** [comment]
Hey @matheuschimelli  how did you get it running? I tried and I always get some erros with react/react-dom x_X

**Comment by matheuschimelli** [comment]
yep, i had the same problem. But for me the fix was very easy. I just change it on package.json line **46:**

the problem:
> "react": "^15.3"

solution:
> "react": "15.3"

i got npm install again and it worked


`

**Comment by matheuschimelli** [comment]
I been thinking for a long time, that maybe this project is dead. It is really sad. But i too see that many issues are created with often, what means that many people still using it for your projects as me. So, what you think of a megaboilerplate community version?

**Comment by matheuschimelli** [comment]
yep, i will create a new issue with this question
